dashboard/src/components/AppView/AppView.tsx

```diff
@@ -97,6 +100,21 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
     if (!newApp) {
       return;
     }
+    if (
+      newApp.chart &&
+      newApp.chart.metadata &&
+      newApp.chart.metadata.name &&
+      newApp.chart.metadata.version &&
+      newApp.chart.metadata.appVersion &&
```

Comment:
what if the chart doesn't have an `appVersion`? We should still be able to check based on `version`.

Reply:
Hmm, well can we match on the fact that `appVersion` is not defined and `version` is? i.e. we'd match with the `azure` repo because in `stable` the `appVersion` is defined. It would be a shame for this whole thing to not work for any chart that does not define the `appVersion`, given it is an optional field in the spec.